mira/dkg/askemo/askemo.json

```diff
@@ -258,7 +280,24 @@
     "physical_min": 0.0,
     "suggested_data_type": "float",
     "suggested_unit": "1 / day",
-    "synonyms": [],
+    "synonyms": [
+      {
+        "type": "referenced_by_symbol",
+        "value": "γ"
+      },
+      {
+        "type": "referenced_by_latex",
+        "value": "\\gamma_1"
+      },
+      {
+        "type": "referenced_by_latex",
+        "value": "\\gamma_2"
+      },
+      {
+        "type": "referenced_by_latex",
+        "value": "\\delta_4"
```

Comment:
Same here, we don't need numbered variants.

Reply:
Done in 11f68a3